framework/src/main/resources/org/fulib/fx/lang/error.properties

```diff
@@ -11,6 +11,7 @@
 1009=Class '%s' annotated with @Title must be a controller or component.
 1010=Method '%s' annotated with @onKey in class '%s' must either take no or exactly one parameter of type KeyEvent.
 1011=Controller '%s' must provide a parent as their view to be able to be shown as a root node.
+1012=%s '%s' in class '%s' annotated with '%s' cannot be accessed as it is private.
```

Comment:
Simpler wording would be `Cannot access private %s '%s' in class '%s' annotated with '%s'`